src/components/common/NavBar.js

```diff
@@ -2,24 +2,47 @@ import React from 'react';
 
 /**
  * Required Props:
- * brandTitle {string} - String representing the website brand name
- * TODO: Include other nav bar links
+ * navBarType {String} - String representing which version of the navbar to render
+ *    Nav Bar Types: 'module', 'homepage', 'catalog', and (??)
+ * 
+ * TODO: Include props for other nav bar information/links
  */
 function NavBar(props) {
-  const { brandTitle } = props;
+  const { navBarType } = props;
+
+  let navBackBtn = undefined;
+  let navBarContainer = "nav-bar-container";
+  const navBarLinks = [];
+
+  if (navBarType === "homepage") {
+    navBackBtn = "hidden";
+    navBarContainer += " homepage-nav-bar";
+    navBarLinks.push("Log In");
+    navBarLinks.push("Get Started");
+  } else if (navBarType === "module") {
+    navBackBtn = "nav-back-btn";
+    navBarContainer += " module-nav-bar";
+    navBarLinks.push("Account");
+  } else if (navBarType === "catalog") {
+    navBackBtn = "hidden";
+    navBarContainer += " homepage-nav-bar";
+    navBarLinks.push("Catalog");
+    navBarLinks.push("Account");
+  }
 
   return (
-    <div className="nav-bar-container">
-      <div className="nav-back-btn">
+    <div className={navBarContainer}>
+      <div className={navBackBtn}>
```

Comment:
Lets avoid adding Nodes just to put a hidden class on them. Instead you can create an element that conditionally renders like this:
```
const backBtn = navBarType === "module" ? (
   <div>...</div>
) : null; 
```